src/opts.c

```diff
@@ -291,6 +291,7 @@ static void print_help(const char *progname) {
 	"    -o relaxed_permissions Disable permissions checks, but only if\n"
 	"                           running neither as UID=0 or GID=0\n"
 	"    -o statfs_omit_ro      do not count blocks of ro-branches\n"
+	"    -o direct_io           Support direct io\n"
```

Comment:
please change "support" to "enable"

Reply:
Before committing, do you know why some programs have trouble if the direct_io option is enabled in a chroot? Bindfs (which I similarly patched) doesn't have this issue.